lib/src/main/scala/spinal/lib/bus/localbus/MinBus.scala

```diff
@@ -0,0 +1,35 @@
+package spinal.lib.bus.localbus
+
+import spinal.core._
+import spinal.lib._
+
+case class MinBusConfig(aw: Int,
+                        dw: Int,
+                        pw: Int = 4,
+                        withStrb: Boolean = false
+                       ){
+  val sw  = dw >> 3
+}
+
+case class MinBus(c: MinBusConfig) extends Interface with IMasterSlave {
```

Comment:
I hope you'll describe the meaning (and timing) of these signals somewhere - its really hard to know what some of them are supposed to be (e.g. the difference between rvld and rdy, what `prot` does).
Apart from that this is a slight extension of AsyncMemoryBus and would maybe be better as some parameters there?

Reply:
`prot `is widely used in the AMBA protocol, integrating the habits of the AMBA protocol to represent secure bits.  and yes , adding some comments would be better

Reply:
I understand that `prot` is used by AMBA - but nobody can know that you are using the same names here compared to AMBA. This bus does in no way show a connection to the AMBA protocols. That why I think it needs at least a description of the intent of the signals + their timing.